contracts/OtokenFactory.sol

```diff
@@ -0,0 +1,177 @@
+pragma solidity =0.6.10;
+
+import {Spawner} from "./packages/Spawner.sol";
+import {IAddressBook} from "./interfaces/IAddressBook.sol";
+import {IWhitelistModule} from "./interfaces/IWhitelistModule.sol";
+
+/**
+ * SPDX-License-Identifier: UNLICENSED
+ * @title A factory for opyn tokens
+ * @author Opyn
+ * @notice Create new otokens and keep track of all created tokens.
+ * @dev Calculate contract address before each creation with CREATE2
+ * and deploy eip-1167 minimal proxies for otoken logic contract.
+ */
+contract OtokenFactory is Spawner {
+    // The Opyn AddressBook contract that records addresses of whitelist module and oToken impl contract.
+    IAddressBook public addressBook;
```

Comment:
Can we rename to AddressBookInterface? easier to remember it is an interface

Reply:
Are you suggesting changing the variable name `addressbook` or the type `IAddressBook`?

If you're talking about the variable, I don't think it's necessary, because all the external contract instance are interfaces. Also, most contracts name their interfaces this way:  [dydx](https://github.com/dydxprotocol/solo/blob/3b6db45cf109fa2cf272843c4192a7d892ad30f4/contracts/protocol/Admin.sol#L95)

If you're talking about the `IAddressBook` type, I don't have strong opinion on this. I know most people just put `I` to represent an interface, so it's kind of a convention. But Compound always uses `Interface`. 

@haythem96  You have any preference?

Reply:
I mostly use `I` at the beginning, feel like adding the `Interface` make it more longer, but yeah it is okay if u guys wanna go with adding the `Interface`.

Reply:
Yes, but Interface makes it more clear to even a newer solidit developer